julia/Project.toml

```diff
@@ -14,6 +13,7 @@ JuliaFormatter = "98e50ef6-434e-11e9-1051-2b60c6c9e899"
 Mustache = "ffc61752-8dc7-55ee-8c37-f3e9cdd09e70"
 Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
 SixelTerm = "65edfddc-f399-4499-8369-a1bd38eee2ea"
+StaticArrays = "90137ffa-7385-5640-81b9-e52037218182"
```

Comment:
This is from merging in #1154.  I'm not exactly sure why this change appears here rather than in #1154, but it resulted from auto-rebuilding the Julia kernels.